firefox-ios/Client/Frontend/Library/Bookmarks/Edit Folder/EditFolderCell.swift

```diff
@@ -17,6 +17,8 @@ class EditFolderCell: UITableViewCell,
         view.addAction(UIAction(handler: { [weak self] _ in
             self?.titleTextFieldDidChange()
         }), for: .editingChanged)
+        let layoutDirection = UIView.userInterfaceLayoutDirection(for: self.semanticContentAttribute)
+        view.textAlignment = layoutDirection == .rightToLeft ? .right : .left
```

Comment:
isn't there a `.natural` alignment so we don't have to figure out layout direction manually??? Does that work?

Reply:
Ahh, TIL - when using [.natural](https://developer.apple.com/documentation/uikit/nstextalignment/natural), the text alignment will be determined by the l10n that the text input is in. So if the text input is in an LTR language, while the app is in a RTL language, the alignment will still be to the left - and I believe this is what is expected. Good callout!

Reply:
I wonder if we should be using this for all text fields...

Reply:
So.... technically, I think that's the default? But sometimes it doesn't behave as we expect because.... Apple, really. But yeah, we should probably be using this.

Reply:
Oh, you're totally right. It works fine in this case - so closing this PR/ticket